### PR TITLE
"Improves" the hypersleep bay console code some more

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
This time, we removed the GLOB.blablaba spam and replace it with list reference.
Should be much more readeable now. It also checks lists fewer times, meaning it should be better with large amounts of gear.
Also, slight typo correction.

Associated list will come later, when I got the energy to learn something new. 
## Why It's Good For The Game
Code improvements
## Changelog
:cl:
code: Improved Hypersleep bay code further
/:cl:

### CODE REVIEW

File: code/game/objects/machinery/cryopod.dm [hunk 1]
```diff
@@ -144,91 +158,53 @@
...
-		GLOB.cryoed_item_list_gun -= I;
-		GLOB.cryoed_item_list_ammo -= I;
-		GLOB.cryoed_item_list_explosive -= I;
-		GLOB.cryoed_item_list_melee -= I;
-		GLOB.cryoed_item_list_clothing -= I;
-		GLOB.cryoed_item_list_food -= I;
-		GLOB.cryoed_item_list_drugs -= I;
-		GLOB.cryoed_item_list_containers -= I;
-		GLOB.cryoed_item_list_other -= I;
+		active_glob_cryo_items_list -= I;
```
Comment (TiviPlus) [inline code/game/objects/machinery/cryopod.dm:197]:
```suggestion
		active_glob_cryo_items_list -= I
```

File: code/game/objects/machinery/cryopod.dm [hunk 2]
```diff
@@ -144,91 +158,53 @@
...
-	GLOB.cryoed_item_list_gun -= I;
-	GLOB.cryoed_item_list_ammo -= I;
-	GLOB.cryoed_item_list_explosive -= I;
-	GLOB.cryoed_item_list_melee -= I;
-	GLOB.cryoed_item_list_clothing -= I;
-	GLOB.cryoed_item_list_food -= I;
-	GLOB.cryoed_item_list_drugs -= I;
-	GLOB.cryoed_item_list_containers -= I;
-	GLOB.cryoed_item_list_other -= I;
+	active_glob_cryo_items_list -= I;
```
Comment (TiviPlus) [inline code/game/objects/machinery/cryopod.dm:206]:
```suggestion
	active_glob_cryo_items_list -= I
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Review by Lumipharon — CHANGES_REQUESTED** [review]
you don't need a var. When I said a single list, I meant a single GLOB list of lists. So instead of all this if(DRUGS) list = blah stuff, you just ref GLOB.CRYOSTUFF[DRUGS]

**Comment by lbnesquik** [comment]
> you don't need a var. When I said a single list, I meant a single GLOB list of lists. So instead of all this if(DRUGS) list = blah stuff, you just ref GLOB.CRYOSTUFF[DRUGS]

Yeah I figured sort of what you meant, I just pushed this out last week to quickly fix those issues.

**Pull request closed without merging**